electron/electron.js

```diff
@@ -25,6 +25,13 @@ function createWindow() {
       ? "http://localhost:3000"
       : `file://${path.join(__dirname, "../dist/index.html")}`
   )
+  tray = new Tray(path.join(__dirname, "./assets/icon.png"))
```

Comment:
Try to declare the variable (**tray**) before initializing it.

Reply:
Declared tray variable